microsoftgraph/microsoft-graph-explorer-v4
# Fix: prevent resize when view expanded

## Overview

If the components are resized and the user clicks the Expand button, the expanded area takes the values of the resize. This PR changes this so that the real estate when the response area is expanded is not affected by the resize action.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Resize so that the response component is quite small
* Run the `my mail` request
* Hit expand. Notice that it's taking up the entire string
* Compare this with what is in staging
